.flake8

```diff
@@ -0,0 +1,6 @@
+[flake8]
+ignore =
+  E501  # line too long, defer to black
+  F401  # unused import, defer to pylint
+  W503  # allow line breaks after binary ops, not after
+  E203  # allow whitespace before ':' (https://github.com/psf/black#slices) 
```

Comment:
Is something missing at the very end of this line? :laughing:

Reply:
fixed